stdpopsim/catalog/HomSap/demographic_models.py

```diff
@@ -1548,3 +1548,136 @@ def _ooa_4pop():
 
 
 _species.add_demographic_model(_ooa_4pop())
+
+
+def _ancient_europe():
+    id = "AncientEurope_4P21"
+    description = "Multi-population model of ancient Europe"
+    long_description = """
+    Population structure that has existed over the last 45,000 years in Europe, leading
+    to modern Europeans. The model demonstrates the divergence of a Basal European
+    Lineages into four ancient populations; Western, Eastern and Caucasus Hunter-
+    Gatherers and Anatolian Farmers. Migration of Anatolian farmers into Western Europe
+    and admixture with Western Hunter-Gatherers produces the European Neolithic Farmers.
+    In West Asia the admixture of Eastern Hunter-Gatherers and Caucasus Hunter-
+    Gatherers leads to the formation of the Yamnaya Steppe population. The Yamnaya
+    migrate into Western Europe to admixture with the Neolithic farmers giving rise to
+    Bronze Age europeans. There is only an exponential growth in population size from
+    then to the Present-day. Samples are taken at multiple point throughout history
+    from each population.
+    """
+    populations = [
+        stdpopsim.Population(
+            id="Pop0",
+            description="1000GenomesEUR/BronzeAge/Neolithic/Anatolian/WestAsian/Basal",
+        ),
+        stdpopsim.Population(id="Pop1", description="Yamnaya/CHG"),
+        stdpopsim.Population(id="Pop2", description="WHG/NorthernEuropean"),
+        stdpopsim.Population(id="Pop3", description="EHG"),
+    ]
+    citations = [
+        stdpopsim.Citation(
+            author="Allentoft et al.",
+            year="in progress",
+            doi="in progress",
```

Comment:
Our defacto policy for model inclusion is that the model appears in a publication. Do you have an idea of when the manuscript is expected to appear? I think a preprint ought to be fine for inclusion here.

Reply:
preprint definitely counts as "publication".